rq/queue.py

```diff
@@ -458,8 +457,19 @@ def enqueue_dependents(self, job, pipeline=None):
                 if pipeline is None:
                     pipe.watch(dependents_key)
 
-                dependent_jobs = [self.job_class.fetch(as_text(job_id), connection=self.connection)
-                                  for job_id in pipe.smembers(dependents_key)]
+                dependent_job_ids = [as_text(_id)
+                                     for _id in pipe.smembers(dependents_key)]
+
+                dependent_jobs = [
```

Comment:
Can we rename this to `jobs_to_enqueue`? Previously all dependent jobs had to be enqueued because there's no multi dependency feature, now we have to do an extra check.

Reply:
0672cd00c6ab5268f83887e66990726904cbe1f9